livekit-agents/livekit/agents/voice_assistant/voice_assistant.py

```diff
@@ -471,21 +478,18 @@ async def _play_speech(self, speech_info: _SpeechInfo) -> None:
 
             self.emit("function_calls_finished", called_fncs)
 
-            tool_calls = []
-            tool_calls_results = []
+            tool_calls_results: list[ChatMessage] = []
 
             for called_fnc in called_fncs:
                 # ignore the function calls that returns None
                 if called_fnc.result is None:
                     continue
 
-                tool_calls.append(called_fnc.call_info)
                 tool_calls_results.append(
-                    llm.ChatMessage.create_tool_from_called_function(called_fnc)
+                    ChatMessage.create_tool_from_called_function(called_fnc)
                 )
 
             chat_ctx = speech_info.source.chat_ctx.copy()
-            chat_ctx.messages.extend(tool_calls)
```

Comment:
@theomonnom Was this correct? Types were wrong

Reply:
ah, fixed it thx.